### PR TITLE
fixing constant check warnings

### LINKED ISSUE #132: Missing docs for archive.R
I think the documentation on `main` was behind. I just updated and force pushed. But I got a slew of warnings for undocumented R6 methods and functions in archive.R. Do we need to add those?

### PR DESCRIPTION
**Pull request by dsweber2**
whenever I run check locally, I get a lot of errors about various documentation bits missing. This fills those in.

Closes #132 apparently.

### CODE REVIEW

File: R/archive.R [hunk 1]
```diff
@@ -451,6 +466,21 @@ epi_archive =
...
+#'   `min_time_value`.)
+#' @param min_time_value Time value specifying the min time value to permit in
+#'   the snapshot. Default is `-Inf`, which effectively means that there is no
+#'   minimum considered.
+#' @param all_versions If `all_versions = TRUE`, then the output will be in
+#'   `epi_archive` format, and contain rows in the specified `time_value` range
+#'   having `version <= max_version`. The resulting object will cover a
+#'   potentially narrower `version` and `time_value` range than `x`, depending
+#'   on user-provided arguments. Otherwise, there will be one row in the output
+#'   for the `max_version` of each `time_value`. Default is `FALSE`.
```
[the remark this answers is not in the document]
Comment (brookslogan) [reply R/archive.R:485]: same for some of the other methods below

[the remark this answers is not in the document]
Comment (dsweber2) [reply R/archive.R:485]: These are primarily about getting check to stop yelling. I just copy-pasted the docs from `methods-epi_archive`; given that you're planning to remove the R6, I don't expect to be actually updating the docs for them outside of this.

[the remark this answers is not in the document]
Comment (brookslogan) [reply R/archive.R:485]: Hmm, somehow I thought they didn't look copy-pasted.  Maybe lined up the wrong methods.  Well, if you do think there's something here that will be lost by the R6 removal, please make a note somewhere.  + maybe note in a comment in this file that these are copy-pasted?

**Review by brookslogan — APPROVED** [review]

**Comment by brookslogan** [comment]
Looks good; see minor request above + please check my tweaks before merging.

**Comment by brookslogan** [comment]
This will close #132.

**Comment by brookslogan** [comment]
Great, you worded that line wrong and now look what happens
```
   Processing triggers for install-info (6.8-4build1) ...
  NEEDRESTART-VER: 3.5
  NEEDRESTART-KCUR: 6.2.0-1018-azure
  NEEDRESTART-KEXP: 6.2.0-1018-azure
  NEEDRESTART-KSTA: 1
  Error: 
  ! error in pak subprocess
  Caused by error in `verify_extracted_package(filename, pkg_cache)`:
  ! 
  '/tmp/RtmpJscovc/file135b521262ed/src/contrib/x86_64-pc-linux-gnu-ubuntu-22.04/4.3/withr_3.0.0.tar.gz'
  is not a valid binary, it is missing withr/Meta/package.rds.
  ---
  Backtrace:
  1. pak::lockfile_install(".github/pkg.lock")
  2. pak:::remote(function(...) { …
  3. err$throw(res$error)
  ---
  Subprocess backtrace:
   1. base::withCallingHandlers(cli_message = function(msg) { …
   2. get("lockfile_install_internal", asNamespace("pak"))(...)
   3. plan$install()
   4. pkgdepends::install_package_plan(plan, lib = private$library, num_workers = nw, …
   5. base::withCallingHandlers({ …
   6. pkgdepends:::handle_events(state, events)
   7. pkgdepends:::handle_event(state, i)
   8. proc$get_result()
   9. processx:::process_get_result(self, private)
  10. private$post_process()
  11. pkgdepends:::install_extracted_binary(filename, lib_cache, pkg_cache, lib, …
  12. pkgdepends:::verify_extracted_package(filename, pkg_cache)
  13. base::throw(pkg_error("{.path {filename}} is not a valid binary, it is missing {…
  14. | base::signalCondition(cond)
  15. global (function (e) …
  Execution halted
  Error: Process completed with exit code 1.
```

**Comment by dsweber2** [comment]
I may have blamed the wrong source of the check errors it seems. Any idea why this is happening? I remember this sort of build error happening like 1/20 times or fewer on other repos, but this is like 1/2 or 3/4...

**Comment by brookslogan** [comment]
Oh, I haven't seen it before!  I'm not sure why.  I'm trying some stuff out locally but no luck getting something similar so far.

Aside: could you please add NEWS.md entries for this + the `select` stuff, or add an Issue reminding us?  We should probably put in some reminder to do this on PR creation.

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
-# epiprocess 0.7.0.9000
+# epiprocess 0.7.1.9999
+
+* Patch `select` for grouped `epi_df`s PR #390
+* switched `other_keys` default from `NULL` to `character(0)` PR #390
+* minor doc updates PR #393
+
+# epiprocess 0.7.0.9999
```
Comment (brookslogan) [inline NEWS.md:7]: We've been using 0.7.0.9999 to mean "development past 0.7.0 that we haven't assigned a real version number yet" & bumping to e.g., 0.7.1 or 0.8.0 on release.  Sound all right?  If so maybe merge with 0.7.0.9999?

Comment (brookslogan) [reply NEWS.md:7]: We're switching to [this version numbering scheme](https://delphi-org.slack.com/archives/C028AUR6MV1/p1705606092907929) so 0.7.1.9999 is good.

**Comment by brookslogan** [comment]
Force push was for styling.  Think this is ready to merge.

**Pull request merged**